scripts/deploy.sh

```diff
@@ -17,5 +17,8 @@ if [[ $TRAVIS_BRANCH == "master" ]]; then
 	mkdir pages
 	cp -R dist/docs/documentation/ pages/documentation
 	cp -R dist/docs/storybook/* pages
+	version=$(node -e 'const package = require("./package.json"); console.log(package.version);')
```

Comment:
Isn't `package.json` version kinda stuck at 0.1.0?

Reply:
nope, since this runs after semantic release `package.json` will have the correct value.